src/Umbraco.Web.Website/AspNetCore/UmbracoWebsiteApplicationBuilderExtensions.cs

```diff
@@ -0,0 +1,24 @@
+using System;
+using Microsoft.AspNetCore.Builder;
+using SixLabors.ImageSharp.Web.DependencyInjection;
+
+namespace Umbraco.Web.Website.AspNetCore
+{
+    public static class UmbracoBackOfficeApplicationBuilderExtensions
+    {
+        public static IApplicationBuilder UseUmbracoWebsite(this IApplicationBuilder app)
+        {
+            if (app == null)
+            {
+                throw new ArgumentNullException(nameof(app));
+            }
+
+
+            // Important we handle image manipulations before the static files, otherwise the querystring is just ignored.
+            app.UseImageSharp();
```

Comment:
We'll need this in the back office too... i know this is early days and thsi is POC work :) just mentioning it. We'll probably have some other common ext methods like UseUmbracoWebCommon. Also on that note, we'll probably end up with another netcore 3.1 class library for common things between the back office and front-end.